job-dsls/jobs/sonarcloud_daily.groovy

```diff
@@ -0,0 +1,153 @@
+/**
+ * Creates jobs running tests with code coverage measurement enabled and reporting results together with code quality
+ * statistics into SonarCloud.
+ */
+import org.kie.jenkins.jobdsl.Constants
+
+def final DEFAULTS = [
+        ghOrgUnit              : Constants.GITHUB_ORG_UNIT,
+        branch                 : Constants.BRANCH,
+        timeoutMins            : 90,
+        ghAuthTokenId          : "kie-ci2-token",
+        label                  : "kie-rhel7 && kie-mem8g",
+        upstreamMvnArgs        : "-B -e -T1C -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true clean install",
```

Comment:
I am not sure, but could we turn off the GWT compilation as this job is for static analysis, we don't need a real GWT compilation?

Reply:
I guess that's right - and the current settings already do it, or is there any other property to be set?

Reply:
sorry, I was just mistakenly reading that option property in the opposite way ;).